ql/src/test/results/clientpositive/llap/annotate_stats_lateral_view_join.q.out

```diff
@@ -503,14 +503,14 @@ STAGE PLANS:
                             Statistics: Num rows: 1 Data size: 376 Basic stats: COMPLETE Column stats: COMPLETE
                             Lateral View Join Operator
                               outputColumnNames: _col0, _col1, _col5, _col6
-                              Statistics: Num rows: 0 Data size: 24 Basic stats: PARTIAL Column stats: NONE
+                              Statistics: Num rows: 0 Data size: 24 Basic stats: PARTIAL Column stats: COMPLETE
```

Comment:
With clone, the following condition is not satisfied since the basic stats of parent operators are PARTIAL.
https://github.com/apache/hive/blob/91e492de239427fc1e38e5e4350cfdce409ebb70/ql/src/java/org/apache/hadoop/hive/ql/optimizer/stats/annotation/StatsRulesProcFactory.java#L2969

Reply:
BTW, it would be better that the UDTF rule puts one in num rows in case that it becomes zero.

Reply:
definetly - I don't think it will be `0` in reality!

Reply:
This is an edge case since `HIVE_STATS_UDTF_FACTOR` is greater than or equal to 1. Anyway, I created a ticket.
https://issues.apache.org/jira/browse/HIVE-24240